openedx/core/djangoapps/content/search/tests/test_handlers.py

```diff
@@ -87,7 +93,16 @@ def test_create_delete_xblock(self, meilisearch_client):
             "block_type": "vertical",
             "context_key": "course-v1:orgA+test_course+test_run",
             "org": "orgA",
-            "breadcrumbs": [{"display_name": "Test Course"}, {"display_name": "sequential"}],
+            "breadcrumbs": [
+                {
+                    "display_name": "Test Course",
+                    "usage_key": "block-v1:orgA+test_course+test_run+type@course+block@course",
```

Comment:
I'm not sure if it's useful or not to include the `usage_key` of the course root block. We generally use the course ID instead in almost all cases. So I kind of lean toward just excluding it. (But leave the sections, subsections, units, etc.)

Reply:
As we removed it from the lib children, I think it makes sense to also not add it for the course level here.
Fixed: 7478009